docs/snapcraft-advanced-features.md

```diff
@@ -127,6 +127,25 @@ will be built after `libpipeline`. Especially if you need specific
 functionality during a build or as part of checks during the `stage` phase,
 this will be handy.
 
+If any part built using the `after` keyword needs to explicitly access
+assets in the stage directory with configuration flags (e.g.; `configure`
```

Comment:
I picked this habit from scientific writing classes for my thesis.

Although they refer to using a comma instead of a semicolon, a separator is required http://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e